.netlify/build/chunks/_astro_assets_D6TztbRf.mjs

```
@@ -0,0 +1,1507 @@
+  };
+  let originalWidth;
+  let originalHeight;
+  let originalFormat;
+  if (options.inferSize && isRemoteImage(resolvedOptions.src) && isRemotePath(resolvedOptions.src)) {
+    const result = await inferRemoteSize(resolvedOptions.src);
+    resolvedOptions.width ??= result.width;
+    resolvedOptions.height ??= result.height;
+    originalWidth = result.width;
+    originalHeight = result.height;
+    originalFormat = result.format;
+    delete resolvedOptions.inferSize;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)